### DISCUSSION TITLE
No Ads

### DISCUSSION THREAD

**Discussion by Fryuni** [dailydotdev/daily#1076] [Feature Requests]
Have some option to be ads free without ripping you off with ad-blockers. I don't mind paying to use the platform without ads, I just want a feed with content that the community upvoted not content that someone paid to be there.

A one-time payment that removes adds for a period (basically without auto-renew) would be nice for the user-friendly aspect, but a normal subscription would be fine

**Comment by idoshamun** [comment]
Hi @Fryuni,

Thanks for the suggestion, we used to have this option but it never took off so we removed it. At this point all the hassle around it doesn't worth the effort but we will reconsider in the future.

**Reply by Fryuni** [reply]
That's too bad. The worst is that my go-to ad-blocker doesn't block ads on daily.dev, so I made a custom solution to remove the ads.

I'm looking forward to some way to support you directly so you're not taking a loss from me.

**Reply by idoshamun** [reply]
Thank you for your support

**Comment by thraizz** [comment]
@idoshamun I'd also love this, just to support your incredible community here on daily.dev beyond sharing it with colleagues 😄 
For a small amount like 4$, i'd love to see no ads and maybe a Supporter badge or something.
Looking for something like this, I only found the merch store. Stickers and merch are cool, but a subscription sounds like a great addition here. Was stripe such an issue that it was not worth the time? I can recommend Paddle from my experience with subscriptions.

**Reply by idoshamun** [reply]
Thanks a lot! We will consider it :)
I also heard good things about Paddle

**Comment by mathieuhays** [comment]
Hello there, just wanted to echo support for the addition of this feature. It's always frustrating to deal with ads especially when they look almost exactly like the actual content. I would love to be able to support daily.dev differently :-)

**Comment by sabdailydev** [comment]
Hi @mathieuhays , @Fryuni , @thraizz , 

We have an ads free version of daily.dev now. It is called daily.dev Plus and comes with several other features. Do give it a look - [Plus](https://app.daily.dev/plus)
